MediaGalleryIntegration/Test/Integration/Plugin/UpdateWysiwygDialogUrlTinyMceTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryIntegration\Test\Integration\Plugin;
+
+use Magento\Framework\ObjectManagerInterface;
+use Magento\Framework\UrlInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\Framework\DataObject;
+use Magento\Tinymce3\Model\Config\Gallery\Config;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Provide integration tests covers update open dialog url functionality for media editor.
+ * @magentoAppArea adminhtml
+ */
+class UpdateWysiwygDialogUrlTinyMceTest extends TestCase
+{
+    private const FILES_BROWSER_WINDOW_URL = 'files_browser_window_url';
+
+    /**
+     * @var ObjectManagerInterface
+     */
+    private $objectManger;
+
+    /**
+     * @var Config
+     */
+    private $tinyMce3Config;
+
+    /**
+     * @var DataObject
+     */
+    private $configDataObject;
+
+    /**
+     * @var string
+     */
+    private $fileBrowserWindowUrl;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp(): void
+    {
+        $this->objectManger = Bootstrap::getObjectManager();
+        $this->tinyMce3Config = $this->objectManger->create(Config::class);
+        $this->configDataObject = $this->objectManger->create(DataObject::class);
+
+        $url = $this->objectManger->create(UrlInterface::class);
+        $this->fileBrowserWindowUrl = $url->getUrl('media_gallery/index/index');
+    }
+
+    /**
+     * Test image open dialog url when enhanced media gallery not enabled.
```

Comment:
I'd add `@magentoConfigFixture default/system/media_gallery/enabled 0` for all tests to keep passing even if enhanced media gallery will be enabled by default